cmd/rest-server/main.go

```diff
@@ -47,6 +47,8 @@ func init() {
 	flags.StringVar(&server.TLSCert, "tls-cert", server.TLSCert, "TLS certificate path")
 	flags.StringVar(&server.TLSKey, "tls-key", server.TLSKey, "TLS key path")
 	flags.BoolVar(&server.NoAuth, "no-auth", server.NoAuth, "disable .htpasswd authentication")
+	flags.BoolVar(&server.NoVerifyUpload, "no-verify-upload", server.NoVerifyUpload,
+		"do not verify the integrity of uploaded data. DO NOT enable unless the rest-server runs on a very low-power decive")
```

Comment:
Typo `decive` --> `device`

```suggestion
		"do not verify the integrity of uploaded data. DO NOT enable unless the rest-server runs on a very low-power device")
```

Reply:
Thanks! Is fixed.